src/main.py

```diff
@@ -98,18 +98,23 @@
     # such as unsafe-inline or unsafe-eval is highly discouraged, as these
     # may lead to XSS attacks.
     'csp_policy': {
-        # https://developers.google.com/fonts/docs/technical_considerations
-        'font-src':    '\'self\' themes.googleusercontent.com '
-                       '*.gstatic.com',
+        # Fallback.
+        'default-src': '\'self\'',
+        # Disallow Flash, etc.
+        'object-src': '\'none\'',
+        # Google Analytics.
+        'script-src': '\'self\' https://www.google-analytics.com/analytics.js',
         # Maps, YouTube provide <iframe> based embedding at these URIs.
-        'frame-src':   '\'self\' www.google.com www.youtube.com',
-        # Assorted Google-hosted APIs.
-        'script-src':  '\'self\' *.googleanalytics.com *.google-analytics.com',
+        'child-src':  '\'self\' https://www.google.com https://www.youtube.com',
+        # Deprecated. Used for supporting browsers that use CSP 1.0 only.
+        'frame-src':  '\'self\' https://www.google.com https://www.youtube.com',
         # In generated code from http://www.google.com/fonts
-        'style-src':   '\'self\' fonts.googleapis.com *.gstatic.com',
-        # Fallback.
-        'default-src': '\'self\' *.gstatic.com',
-        'report-uri':  '/csp',
+        'style-src':  '\'self\' \'unsafe-inline\' https://fonts.googleapis.com '
```

Comment:
This is primarily because style/expression based XSS isn't possible in newer versions of IE, right?  Can we add a comment to that effect (ideally incorporating a reference to some public statistics about browser distribution as above)?